contracts/syndicate/DaiBackstopSyndicate.sol

```diff
@@ -131,45 +127,56 @@ contract DaiBackstopSyndicate is IDaiBackstopSyndicate, SimpleFlopper, ERC20 {
   /// @notice Triggers syndicate participation in an auction, bidding 50k DAI for 500 MKR
   /// @param auctionId ID of the auction to participate in
   function enterAuction(uint256 auctionId) external {
-    // Ensure that the auction in question has not already been entered
     require(
-      _bidders[auctionId] == address(0x0),
-      "DaiBackstopSyndicate/enterAuction: Already participating in this auction"
+      !_activeAuctions.contains(auctionId),
+      "DaiBackstopSyndicate/enterAuction: Auction already active"
     );
 
-    // Create auction's Bidder contract and approve it for VAT
-    Bidder bidder = new Bidder(SimpleFlopper.getFlopperAddress(), auctionId);
-    _bidders[auctionId] = address(bidder);
-    _VAT.hope(address(bidder));
+    // dai has 45 decimal places
+    (uint256 amountDai, , , , ) = SimpleFlopper.getCurrentBid(auctionId);
+
+    // lot needs to have 18 decimal places, and we're expecting 1 mkr == 100 dai
+    uint256 expectedLot = (amountDai / 1e27) / 100;
 
-    // Submit Bid. Should revert if bid is invalid
-    bidder.submitBid();
+    // Place the bid, reverting on failure.
+    SimpleFlopper._bid(auctionId, expectedLot, amountDai);
 
-    // Prevent further deposits
+    // Prevent further deposits.
     if (_status != Status.ACTIVATED) {
       _status = Status.ACTIVATED;
     }
 
-    // Register auction if successful participation
+    // Register auction if successful participation.
     _activeAuctions.add(auctionId);
 
     // Emit an event to signal that the auction was entered.
-    emit AuctionEntered(auctionId, address(bidder));
+    emit AuctionEntered(auctionId, expectedLot, amountDai);
   }
 
   // Anyone can finalize an auction if it's ready
   function finalizeAuction(uint256 auctionId) external {
-    // Determine the bidder for the auction in question.
-    Bidder bidder = Bidder(_bidders[auctionId]);
+    require(
+      _activeAuctions.contains(auctionId),
+      "DaiBackstopSyndicate/enterAuction: Auction already finalized"
+    );
+
+    // If auction was finalized, end should be 0x0.
+    (,, address bidder,, uint48 end) = SimpleFlopper.getCurrentBid(auctionId);
 
-    // Finalize auction from bidder, sending Dai or MKR back to this contract.
-    bidder.finalize();
+    // If auction isn't closed, we try to close it ourselves
+    if (end != 0) {
+      // If we are the winning bidder, we finalize the auction
+      // Otherwise we got outbid and we withdraw DAI
```

Comment:
Update comment since we don't need to withdraw DAI back to syndicate, we should just "close" that auction